src/plugins/navigation_embeddable/public/components/dashboard_link/dashboard_link_component.tsx

```diff
@@ -86,10 +86,10 @@ export const DashboardLinkComponent = ({
       };
     }
     return {
-      tooltipTitle: Boolean(dashboardDescription) ? dashboardTitle : undefined,
-      tooltipMessage: dashboardDescription || dashboardTitle,
+      tooltipTitle: Boolean(dashboardDescription) ? linkLabel : undefined,
+      tooltipMessage: dashboardDescription || linkLabel,
     };
-  }, [error, dashboardTitle, dashboardDescription]);
+  }, [error, linkLabel, dashboardDescription]);
```

Comment:
Figured I might as well quickly make this change per the offline tooltip description - previously, we were always showing the saved object title in the tooltip; now, we default to the custom label, if provided. This means that the link label and the tooltip will **always** display the same value